partner_event/models/res_partner.py

```diff
@@ -0,0 +1,56 @@
+
+# © 2014 Tecnativa S.L. - Pedro M. Baeza
+# © 2015 Tecnativa S.L. - Javier Iniesta
+# © 2016 Tecnativa S.L. - Antonio Espinosa
+# © 2016 Tecnativa S.L. - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    registrations = fields.One2many(
+        string="Event registrations",
+        comodel_name='event.registration', inverse_name="attendee_partner_id")
+    event_count = fields.Integer(
+        string='Events',
+        compute='_compute_event_count',
+        help="Count of events with confirmed registrations.",
+    )
+    registration_count = fields.Integer(
+        string='Event registrations number',
+        compute='_compute_registration_count',
+        store=True)
+    attended_registration_count = fields.Integer(
+        string='Event attended registrations number',
+        compute='_compute_attended_registration_count', store=True)
+
+    @api.multi
+    @api.depends('registrations')
+    def _compute_registration_count(self):
+        for partner in self:
+            partner.registration_count = len(partner.registrations)
+
+    @api.multi
+    def _compute_event_count(self):
```

Comment:
:fire: This method [exists upstream](https://github.com/odoo/odoo/blob/2cd730998bd1630a4386607186290c7257df39cd/addons/event/models/res_partner.py#L11-L13).